### PR TITLE
Splitted swap and memory graph into two different graphs

### PR DESCRIPTION
**Pull request by Noodlesalat**
According to #41 I have changed the code to split swap and memory graphs for a better overview.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -86,42 +87,57 @@
...
 			memoryUsageChart.streamTo(document.getElementById("memorycanvas"), 1000/*delay*/);
 			memoryUsageLine = new TimeSeries();
 			memoryUsageChart.addTimeSeries(memoryUsageLine, {lineWidth:1, strokeStyle:'rgb(0, 255, 0)', fillStyle:'rgba(0, 255, 0, 0.2)'});
+			swapUsageLine = new TimeSeries();
+      memoryUsageChart.addTimeSeries(swapUsageLine, {lineWidth:1, strokeStyle:'rgb(255, 0, 0)', fillStyle:'rgba(255, 0, 0, 0.2)'});
 		}
 
 		$('#memFooterInfo').text(t('serverinfo', 'Total')+": "+OC.Util.humanFileSize(memTotalBytes)+" - "+t('serverinfo', 'Current usage')+": "+OC.Util.humanFileSize(memUsageBytes));
 		memoryUsageLine.append(new Date().getTime(), memUsageGB);
+		$('#swapFooterInfo').text("Total (SWAP): "+OC.Util.humanFileSize(swapTotalBytes)+" - Current usage: "+OC.Util.humanFileSize(swapUsageBytes));
```
Comment (patschi) [inline js/script.js]: Add "Current usage" into `t()` JS-function to translate the string like two lines above at `#memFooterInfo`. Further I'd suggest format like:
"RAM: Total: X GB - Current usage: X GB
SWAP: Total X GB - Current usage: X GB"

File: js/script.js [hunk 2]
```diff
@@ -86,42 +87,57 @@
...
+				maxValue:maxValueOfChart,
+				minValue:0,
 				grid:{fillStyle:'rgba(0,0,0,0.03)',strokeStyle:'transparent'},
 				labels:{fillStyle:'rgba(0,0,0,0.4)', fontSize:12}
 			});
 			memoryUsageChart.streamTo(document.getElementById("memorycanvas"), 1000/*delay*/);
 			memoryUsageLine = new TimeSeries();
 			memoryUsageChart.addTimeSeries(memoryUsageLine, {lineWidth:1, strokeStyle:'rgb(0, 255, 0)', fillStyle:'rgba(0, 255, 0, 0.2)'});
+			swapUsageLine = new TimeSeries();
+      memoryUsageChart.addTimeSeries(swapUsageLine, {lineWidth:1, strokeStyle:'rgb(255, 0, 0)', fillStyle:'rgba(255, 0, 0, 0.2)'});
```
Comment (patschi) [inline js/script.js]: Missing two tabs here :)

File: js/script.js (hunk 1 above)
Comment (patschi) [inline js/script.js]: Further I'd suggest format like:
[  ] RAM: Total: X GB - Current usage: X GB
[  ] SWAP: Total X GB - Current usage: X GB
Where "[  ]" is any sort of little square representing the used color in the line chart.

**Comment by patschi** [comment]
LGTM

image

**Comment by Noodlesalat** [comment]
Hey, are there any reasons for not reviewing? :)

**Comment by MorrisJobke** [comment]
One thing (that's why drone fails): Could you add the sign-off message to every commit: https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work

Thanks :)

**Pull request closed without merging**